mgsr/index.html

```diff
@@ -110,34 +110,57 @@ <h2>Team Up Bot Game Results Tool</h2>
     <form id="mainform" class="fairway" onsubmit="return false;">
         <div id="1" onchange="updateState()">
         <label for="placement">Place:</label>
-            <input class="place" type="number" min="1" max="4" value="1" id="placement1" list="placements">
```

Comment:
The `<datalist>` approach seems to have a bad UX for number spinners in mobile -- use standard select-based spinner instead